src/main/java/com/akproject/WebLinkChecker/controllers/ProjectController.java

```diff
@@ -0,0 +1,75 @@
+package com.akproject.WebLinkChecker.controllers;
+
+import com.akproject.WebLinkChecker.model.Project;
+import com.akproject.WebLinkChecker.service.ProjectService;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+//@RequestMapping(path = "/weblinkcheker")
+@RequiredArgsConstructor
+public class ProjectController {
+
+    @NonNull
+    private ProjectService projectService;
+
+    /**
```

Comment:
1) Если хочешь писать джава доки, то почитай как они нормально должны выглядеть (@param, @throws, @since и т.д.) и пиши их на английском
2) Если не хочешь писать нормальные джавадоки, то не пиши их вообще